### ISSUE TITLE
Any Ideas for better Debugging Support in the Browser?

### ISSUE THREAD

**Issue by jawil003** [RelaxedJS/ReLaXed#185]
I like to debug my Pages in Chrome when anything is wrong. The Issue happens with the Viewport Size and multiple Pages? Is there any Solution to create a DIN A4 like Preview in the Browser? Anyone knows anything like that? Or a way to develop this as an Extension or something like it?

**Comment by DanielRuf** [comment]
You can emulate the `print` media in the developer tools of Chrome.

As we use Puppeteer it is just a normal headless Chromium instance.

**Comment by jawil003** [comment]
Ok that's great news. Do you have a tutorial or link how I could emulate that? Thought about changing the viewport of HTML but I'm not sure if that is possible. Totally beginner here 😅

**Comment by DanielRuf** [comment]
I think https://developers.google.com/web/tools/chrome-devtools/css/print-preview should be helpful.

**Comment by jawil003** [comment]
Yeah but this doesn't prevent Pages from floating into each other which makes it harder to debug
